cmd/injector/options/options.go

```diff
@@ -31,14 +30,16 @@ type Options struct {
 	Metrics     *metrics.Options
 }
 
+var log = logger.NewLogger("dapr.injector.options")
+
 func New() *Options {
 	var opts Options
 
 	flag.IntVar(&opts.HealthzPort, "healthz-port", 8080, "The port used for health checks")
 
-	flag.StringVar(&credentials.RootCertFilename, "issuer-ca-secret-key", credentials.RootCertFilename, "Certificate Authority certificate secret key")
-	flag.StringVar(&credentials.IssuerCertFilename, "issuer-certificate-secret-key", credentials.IssuerCertFilename, "Issuer certificate secret key")
-	flag.StringVar(&credentials.IssuerKeyFilename, "issuer-key-secret-key", credentials.IssuerKeyFilename, "Issuer private key secret key")
+	depCAFlag := flag.String("issuer-ca-secret-key", "", "DEPRECATED; Certificate Authority certificate secret key")
```

Comment:
I think these flags can be deleted right away.

This is a control plane service that is only used in K8s. There's no risk that someone accidentally passes these flags, since the injector is only deployed by Helm.

Reply:
I'm happy to do this! I'll make the change.